src/dodal/devices/i15/focussing_mirror.py

```diff
@@ -0,0 +1,71 @@
+from ophyd_async.core import StandardReadable
+from ophyd_async.epics.motor import Motor
+
+
+class FocusingMirrorBase(StandardReadable):
+    """Focusing Mirror with curve, ellip & pitch"""
+
+    def __init__(
+        self,
+        prefix: str,
+        name: str = "",
+    ):
+        with self.add_children_as_readables():
+            self.curve = Motor(prefix + "CURVE")
+            self.ellipticity = Motor(prefix + "ELLIP")
+            self.pitch = Motor(prefix + "PITCH")
+
+        super().__init__(name)
+
+
+class FocusingMirrorHorizontal(FocusingMirrorBase):
+    """Focusing Mirror with curve, ellip, pitch & X"""
+
+    def __init__(
+        self,
+        prefix: str,
+        name: str = "",
+    ):
+        with self.add_children_as_readables():
+            self.x = Motor(prefix + "X")
+
+        super().__init__(prefix, name)
```

Comment:
Could: Are you aware of the declarative way of defining devices (https://blueskyproject.io/ophyd-async/main/explanations/decisions/0009-procedural-vs-declarative-devices.html) I'm personally not a massive fan as it's a bit too magic for me but it would reduce all this down to 1 line so maybe you would prefer it

Reply:
Thanks, I'll have a look.